### PR TITLE
Add micro:bit tutorial

### PR DESCRIPTION
**Pull request by gregersrygg**
How to get started with NB-IoT and a micro:bit

### CODE REVIEW

**Comment by gregersrygg** [comment]
@stalehd asked me to merge. Will still fix review comments, so please review anyways.

File: docs/tutorials/microbit-basic.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+P1 - orange | TXD
+
+### Edge connector with pin headers
+The recommended method to connect a micro:bit to EE-NBIOT-01
+<a href="img/microbit-edge-connector.jpg" target="_blank">![Using an edge connector with pin headers](img/microbit-edge-connector-small.jpg)<br/>
+See image in full resolution</a>
+
+
+### Improvise wire connection
+If you don't have an edge connector nearby and don't have time to wait for one, check out [this guide](https://lorrainbow.wordpress.com/2017/11/11/how-to-connect-wires-to-the-microbit/). Or improvise using what you have available.
```
Comment (aschultzen) [inline docs/tutorials/microbit-basic.md:43]: Just nitpicking about the formatting:

> If you don’t have an edge connector nearby and don’t have time to wait for one, check out this guide. Or improvise using what you 
> *IMAGE*
> have available.

File: docs/tutorials/microbit-basic.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+1. The micro:bit should show up as a regular USB memory stick on your computer, called MICROBIT
+1. Click the ![big download](img/makecode-download.png) button to download the sketch and save it to the MICROBIT drive
+1. The sketch will now magically be flashed to the micro:bit
+1. After a few seconds you should see «PRESS A/B» scroll by on the LEDs on the micro:bit
+1. Get a pen and paper, press the A/B buttons and write down the numbers
+1. Follow our [Getting started with the Telenor NB-IoT Developer Platform guide](getting-started.html) and use the IMEI/IMSI from the step above
+1. Leave the browser tab with live device data open, so we can see data we send in the next section
+
+## Install nbiot extension
+1. Go to [makecode.microbit.org](https://makecode.microbit.org/)
```
Comment (aschultzen) [inline docs/tutorials/microbit-basic.md:60]: What is makecode? It's never been mentioned before in this document.

File: docs/tutorials/microbit-basic.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    - ![nbiot menu](img/makecode-nbiot-menu.png)
+
+
+## Hello World
+Drag-n-drop blocks from the menu to reproduce the blocks below:
+
+![Hello World example](img/microbit-hello.png)
+
+### What's going on here?
+In the block editor, every chain of blocks starts with an event (something happened). All the blocks inside of it will be executed in order from top to bottom.
```
Comment (aschultzen) [inline docs/tutorials/microbit-basic.md:78]: The (something happened) part makes perfect sense to me, but perhaps you could elaborate a bit one it? Like. what's the alternative?

File: docs/tutorials/microbit-basic.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+![Hello world in NB-IoT Developer Platform](img/payload-hello.png)
+
+## Sending useful data
+Sending a hard-coded text isn't that useful other than to check that everything works as it should. The fun really begins when we start connecting the physical world with the digital. So try to send some sensor data (like temperature), or send a message when something happens (like a button press or that the micro:bit moves). The micro:bit has a few sensors built in, but if you feel adventorous you can also add other sensors by connecting to one of the pins (definitely get an edge connector breakout board for that). Remember that NB-IoT is meant for small data amounts, so try to keep the size of the data small and don't send too frequently.
```
Comment (aschultzen) [inline docs/tutorials/microbit-basic.md:87]: Nit picking really, but isn't it spelled adventurous? Or am I sparking the whole color vs colour debate?

File: docs/tutorials/microbit-basic.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+Here are some examples of what you can do:
```
Comment (aschultzen) [inline docs/tutorials/microbit-basic.md:89]: I love the examples!

**Review by aschultzen — COMMENTED** [review]
Just a couple of smaller things, nice work!

File: docs/tutorials/microbit-basic.md (hunk 2 above)
Comment (gregersrygg) [reply docs/tutorials/microbit-basic.md:60]: It was actually mentioned under **Register device**. Can you please be a bit more specific in what needs to be explained?

File: docs/tutorials/microbit-basic.md (hunk 3 above)
Comment (gregersrygg) [reply docs/tutorials/microbit-basic.md:78]: What _is_ the alternative? Please elaborate 🙂

File: docs/tutorials/microbit-basic.md (hunk 2 above)
Comment (aschultzen) [reply docs/tutorials/microbit-basic.md:60]: My bad. Nevermind!

File: docs/tutorials/microbit-basic.md (hunk 3 above)
Comment (aschultzen) [reply docs/tutorials/microbit-basic.md:78]: A procedure for instance.

File: docs/tutorials/microbit-basic.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+### Temperature logger
+Log data at a regular interval. Make sure you don't send too much data too often, or you might run out of data quota.
+
+![Temperature logger example blocks](img/microbit-temp-logger.png)
```
Comment (gordonklaus) [inline docs/tutorials/microbit-basic.md:94]: This image is a smaller and harder to read than the Hello World one.  Maybe also add a link to see it in full resolution

File: docs/tutorials/microbit-basic.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+### Sending a message when _something happens_
+To reduce power and data consumption, you can choose to only send data when _something happens_. You read a sensor value and send an alert when the value is not within the expected range. What sensor to use depends of the use-case, but you can connect almost any sensor that runs on 3.3V to the micro:bit, and many of the most popular ones have a micro:bit extension already.
+
+Say you have a fridge or freezer with food worth a lot of money in a building you don't visit that often. If the fridge/freezer suddenly stops working you'd like to know as soon as possible. Here is one way that could be solved.
+
+![Example code for fridge alert](img/microbit-fridge-alert.png)
```
Comment (gordonklaus) [inline docs/tutorials/microbit-basic.md:101]: This image is also small and hard to read.

**Review by gordonklaus — COMMENTED** [review]
Very nice!

File: docs/tutorials/microbit-basic.md (hunk 5 above)
Comment (gregersrygg) [reply docs/tutorials/microbit-basic.md:89]: Thanks 😊

**Pull request merged**